src/app/component/ProductActions/ProductActions.component.js

```diff
@@ -194,13 +195,33 @@ export default class ProductActions extends PureComponent {
         );
     }
 
+    renderAddToButtons() {
+        const {
+            product,
+            quantity,
+            configurableVariantIndex
+        } = this.props;
+
+        return (
+            <>
+                <ProductWishlistButton
```

Comment:
FYI (due this being moved out of here due design). Make sure, to move this into separate method.